Cannot install on Ubuntu 20.04

I'm having problem installing on my system:

```
% gnuplot --version
gnuplot 5.2 patchlevel 8

% lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.1 LTS
Release:        20.04
Codename:       focal

% zef install Chart::Gnuplot
===> Searching for: Chart::Gnuplot
===> Searching for missing dependencies: Distribution::Builder::MakeFromJSON
===> Searching for missing dependencies: System::Query
===> Building: Chart::Gnuplot:ver<0.0.19>:auth<cpan:TITSUKI>
[Chart::Gnuplot] configure: error:     texdir is not given and there is no kpsexpand, please tell where to install
[Chart::Gnuplot] The spawned command './configure --prefix=/home/luca/.p6chart-gnuplot' exited unsuccessfully (exit code: 1, signal: 0)
[Chart::Gnuplot]   in method install-gnuplot at /home/luca/.zef/store/Chart-Gnuplot-0.0.19.tar.gz/Chart-Gnuplot-0.0.19/lib/Chart/Gnuplot/CustomBuilder.pm6 (Chart::Gnuplot::CustomBuilder) line 44
[Chart::Gnuplot]   in method build at /home/luca/.zef/store/Chart-Gnuplot-0.0.19.tar.gz/Chart-Gnuplot-0.0.19/lib/Chart/Gnuplot/CustomBuilder.pm6 (Chart::Gnuplot::CustomBuilder) line 20
[Chart::Gnuplot]   in block <unit> at -e line 1
===> Building [FAIL]: Chart::Gnuplot:ver<0.0.19>:auth<cpan:TITSUKI>

```

I tin is something tied to `TeX`, but I don't understand what it is.

--- a/lib/Chart/Gnuplot/CustomBuilder.pm6
+++ b/lib/Chart/Gnuplot/CustomBuilder.pm6
@@ -41,7 +41,7 @@ class Chart::Gnuplot::CustomBuilder:ver<0.0.19> is Distribution::Builder::MakeFr
         my $extractor = Zef::Extract.new(:backends(@extract-backends));
         my $extract-dir = $extractor.extract(Candidate.new(:uri($archive-file)), $*CWD);
         chdir("gnuplot-5.2.6");
-        shell("./configure --prefix=$prefix");
+        shell("./configure --prefix=$prefix --with-latex --with-texdir={$prefix}/share/texmf/tex/latex/gnuplot");
         shell("make");
         shell("make install");
         chdir($goback);

